extension-types.md

```diff
@@ -38,18 +38,29 @@ metadata).
 ## Extension metadata
 
 When GeoArrow-encoded Arrays have the `ARROW:extension:metadata` metadata
-field set, it must be seriaized as a UTF-8 encoded JSON object. The following
-keys in the JSON metadata object are supported:
-
-- `crs`: A JSON object describing the coordinate reference system (CRS)
-  using [PROJJSON](https://proj.org/specifications/projjson.html).
-  This key can also be omitted if the producer does not have any
-  information about the CRS. Note that regardless of the axis
-  order specified by the CRS, axis order will be interpreted
+field set, it must be seriaized as a UTF-8 encoded JSON object. The extension
+metadata specification is intentionally aligned with the
+[GeoParquet column metadata specifification](https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#metadata).
+The following keys in the JSON metadata object are supported:
+
+- `crs`: One of:
+
+    - A JSON object describing the coordinate reference system (CRS)
+      using [PROJJSON](https://proj.org/specifications/projjson.html).
+    - A string containing an undefined CRS representation. This option
+      is intended as a fallback for producers (e.g., database drivers or
+      file readers) that are provided a CRS in some form but do not have the
+      means by which to convert it to PROJJSON.
```

Comment:
If it's either JSON or a random string, how do you know whether to try to parse it as json?

Reply:
If we're going to allow _anything_ I think it would be a lot easier to have different keys for different value types

Reply:
Sorry I forgot that this was already inside a JSON object. I thought for a second that it was a top-level key in the Arrow extension metadata